common/flatpak-installation.c

```diff
@@ -1440,6 +1440,27 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_min_free_space:
+ * @self: a #FlatpakInstallation
+ *
+ * A convenient wrapper around ostree_repo_get_min_free_space.
+ *
+ * Returns: Value (in bytes) of min-free-space-* config option of
+ * the Ostree repo.
```

Comment:
Various problems/typos in this documentation comment. Suggested replacement:
```
/**
 * flatpak_installation_get_min_free_space_bytes:
 * @self: a #FlatpakInstallation
 *
 * A convenience wrapper around ostree_repo_get_min_free_space_bytes().
 *
 * Returns: Value (in bytes) of the `min-free-space-*` config options of the OSTree repo
 * Since: 1.0.2 (or whatever upstream release this ends up in)
 */
```